native_client/ctcdecode/scorer.cpp

```diff
@@ -22,12 +22,14 @@
 #include "lm/state.hh"
 #include "util/string_piece.hh"
 
+#include "fst/script/convert.h"
+
 #include "decoder_utils.h"
 
 using namespace lm::ngram;
 
-static const int MAGIC = 'TRIE';
-static const int FILE_VERSION = 3;
+static const int32_t MAGIC = 'TRIE';
+static const int32_t FILE_VERSION = 3;
```

Comment:
And so the file itself remains the same ?

Reply:
Well, making this int in the first place was a mistake. As far as I remember most common architectures have 32-bit int, but there's a chance someone could have generated tries with some obscure system with some other int size, and then they update to this commit, and try to load the file, and that would break.

Reply:
@reuben sorry, I was referring to the `ConstFst` changes, it does not change the layout of the file and we don't have to bump the version ?

Reply:
Oh, duh. It does change the file and we do need to bump it :)